src/wrappers/DappProvider/DappProvider.tsx

```diff
@@ -39,7 +46,11 @@ export const DappProvider = ({
   if (externalProvider != null) {
```

Comment:
this logic here has if elseif. Can stay like this or have a small helper above that uses early returns